challenges/classes.js

```diff
@@ -1,7 +1,62 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+// function CuboidMaker(attributes){
+//     this.length = attributes.length;
+//     this.width = attributes.width;
+//     this.height = attributes.height;
+//   };
+
+class cuboidMaker{
+    constructor(attrs){
```

Comment:
It's not required, but it is a pretty strong convention that classes are capitalized.